bin/packages/watch.js

```diff
@@ -59,7 +59,9 @@ function isDirectory( pathname ) {
 function isSourceFile( filename ) {
 	// Only run this regex on the relative path, otherwise we might run
 	// into some false positives when eg. the project directory contains `src`
-	const relativePath = path.relative( process.cwd(), filename );
+	const relativePath = path
+		.relative( process.cwd(), filename )
+		.replace( /\\/g, '/' );
 
```

Comment:
On Windows, `filename` parameter includes  backslashes.
This function always return `false` on Windows because the regular expression is written with forward slashes.
Therefore, I replaced it with forward slashes.

Reply:
Looks familiar 😄